src/data/categories/readExcel.js

```diff
@@ -0,0 +1,62 @@
+import XLSX from "xlsx";
+import axios from "axios";
+
+const fetchData = async (url) => {
+  const data = await axios({
+    method: "get",
+    url: url,
+    responseType: "blob",
+  })
+    .then((res) => res.data)
+    .catch((err) => console.log(err));
```

Comment:
console.log is fine for development but might want to consider removing it once the product is ready and replaced it with something more meaningful.

Reply:
perhaps redirecting to an error page would be better? currently I am editting the error page to display "Coming Soon". perhaps having both error page and coming soon page would be good. what do you think?